README.md

```diff
@@ -1,10 +1,12 @@
-# Gubernator
-
-Gubernator is a distributed, high performance, cloud native and stateless rate
-limiting service.
+<h2 align="center">
```

Comment:
Interesting you used HTML.  Why not Markdown?

Reply:
I copied it from another repo with a fancy readme. If there another way to use markdown to embed an image and have it centered, we should do that instead.